src/index.ts

```diff
@@ -250,6 +250,46 @@ function strEncoder(data: Buffer): string {
   return encodeEd25519PublicKey('ed25519PublicKey', data)
 }
 
+function tezosAddressEncoder(data: Buffer): string {
+  if (data.length !== 22 && data.length !== 21) { throw Error('Unrecognised address format'); }
+
+  let prefix: Buffer;
+  switch (data.readUInt8(0)) {
+    case 0x00:
```

Comment:
Is this version byte canonical, or something we're adding? If it's something we're adding, are the two address formats incompatible, or is there a tz1/tz2/tz3 address for each KT1 address?

Reply:
it was implemented by the original contributor so I am not certain but it looks the version byte is mentioned here

https://tezos.stackexchange.com/questions/183/base58-encoding-decoding-of-addresses-in-micheline